### PR TITLE
Add ORO Pool

### PR DESCRIPTION
**Pull request by ORO-STAKE-POOL**

### CODE REVIEW

File: adapools.json [hunk 1]
```diff
@@ -582,7 +582,12 @@
         "pool_id": "68831f852f4d299157f0911eb74746b11b8bcc65c07f3ba7ea136910",
         "member_since": "2021-03-25",
         "name": "RHEA"
+      },
+      "113": {
+        "pool_id": "4d827f2056879bce6a78963e1feba4825d80dc8a0143db3165931a92",
+        "member_since": "2021-03-26",
+        "name": "ORO"
       }
```
Comment (SinglePoolAlliance) [inline adapools.json:590]: adapools.json is automatically updated! 

Please no manual modifications

**Review by SinglePoolAlliance — APPROVED** [review]

File: registry.json [hunk 2]
```diff
@@ -1600,5 +1600,20 @@
...
+  "poolId": "4d827f2056879bce6a78963e1feba4825d80dc8a0143db3165931a92", 
+  "social": {                                                           
+  	"facebook": "", 
+  	"twitter": "@OroPool",
+  	"website": "http://www.oroops.com/",     
+  	"reddit": "",  
+  	"instagram": "",
+  	"youtube": ""
+  },
+  "telegram": "oro_pool",
```
Comment (lophre) [inline registry.json:1615]: You can't use a channel name in the telegram field.  You need to provide us with a valid telegram username as voting is done using your telegram account.

Comment (ultron001) [reply registry.json]: Updated to username

**Review by ultron001 — COMMENTED** [review]
Updated telegram username

**Comment by lophre** [comment]
Merged manually to resolve conflicts.

**Pull request closed without merging**